src/dashboard/src/templates/administration/processing_edit.html

```diff
@@ -0,0 +1,51 @@
+{% extends "layout_fluid.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Administration" %}{% endblock %}
+{% block h1 %}{% trans "Administration" %}{% endblock %}
+{% block page_id %}Administration{% endblock %}
+
+{% block content %}
+
+  <div class="row">
+
+    {% include "administration/sidebar.html" %}
+
+    <div class="span12">
+
+      <h3>{% trans "Processing configuration" %}</h3>
+      
+      {% if new %}
+        <form class="form-stacked" method="post">
+      {% else %}
+        <form class="form-stacked" method="post">
+      {% endif %}
+
+        {% csrf_token %}
+
+        <table>
+
+          {% for field in form %}
+            <tr>
+              <td>
+                {{ field.label_tag }}
+                {{ field.errors }}
+              </td>
+              <td>{{ field }}</td>
+            </tr>
+          {% endfor %}
+
+        </table>
```

Comment:
Other options for display include using the _form template ,or just {{form}}, but this look fine too.